src/Mailgun/Resource/Api/SimpleResponse.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright (C) 2013-2016 Mailgun.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+namespace Mailgun\Resource\Api;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class SimpleResponse
```

Comment:
It sure about this class. But I understand the need for it...

Does it miss httpResponseCode?